tools/code-generation/smithy/codegen/cpp-smoke-tests-codegen/src/main/java/com/amazonaws/util/awsclientsmithygenerator/generators/SmithyC2JNamespaceMap.java

```diff
@@ -0,0 +1,55 @@
+package com.amazonaws.util.awsclientsmithygenerator.generators;
+
+import java.lang.reflect.Type;
+import java.util.Map;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+public class SmithyC2JNamespaceMap {
+    private static SmithyC2JNamespaceMap instance = null;
+    private  static Map<String, String> SMITHY_C2J_SERVICE_NAME_MAP;
```

Comment:
this really isnt static because every constructor initializes it, might as well make it a member variable